docs/_static/custom.css

```diff
@@ -25,3 +25,23 @@
 .toggle{
   background: #FBFBFB;
 }
+
+article.pytorch-article .method dt {
+    border-left: none;
+    border-top: 3px solid var(--purple);
+    padding-left: 4em;
+    padding-right: 100px;
+}
+
+article .method dt {
+    background: #f3f4f7;
+    word-wrap: break-word;
+}
+
+article .method dd {
+    padding-left: 3em;
+}
+
+.wy-table-responsive {
+    max-height: unset;
+}
```

Comment:
I would like to stress that even though I am fine with keeping the style consistent, **this** particular block I find very important. If there is no other way of achieving the same, I would highly argue to keep this in here.

This block removes the ability to scroll inside of tables which list e.g. the classes in a module doc page. I find this particularly important, because without it **there is NO visual indicator** that the table is actually scrollable in the first place. E.g. in the following screenshot (from the current `main` docs), a user will only learn about the `transformers` module, if they accidentally scroll within the table:
![screenshot_1667373268](https://user-images.githubusercontent.com/21973473/199424037-e1e0c4ab-53b0-4f66-9993-07bc07ed14eb.png)

Reply:
I am not sure about the scrollable - when I look at things they all have horiz and vert scrollbars and with different tables on the same page they are all different widths which does look ugly. We have the same css as Terra yet it seems the outcome is different. Here is something from Terra where you can see it has scroll bars (even if not needed) and the widths are different with the highlighted bars
![image](https://user-images.githubusercontent.com/40241007/199500087-35efe04f-67d7-4a89-9d26-03d7a7076daf.png)




Things were setup so we had the same look/style for docs, we use same css same these etc - I am not sure why things are different - maybe we need to tweak something elsewhere perhaps in conf.py @manoelmarques any chance of taking a look.

Reply:
In that case it may be my browser because the same snippet does not have scroll bars for me:
![screenshot_1667395446](https://user-images.githubusercontent.com/21973473/199500947-81562d70-6354-46cb-b5af-64e517a666a3.png)

Reply:
This is the [current published Nature ](https://qiskit.org/documentation/nature/apidocs/qiskit_nature.html) that looks the same - all sections have scroll bars including the long sub modules one

![image](https://user-images.githubusercontent.com/40241007/199500998-a164c56e-497a-4166-8898-34db68261435.png)

Reply:
this difference is most likely because of the browser. I use chrome which doesn't show those scroll bars but if you're looking in safari/firefox maybe it does?

Reply:
It could be browser specific - I am using Firefox - not good that something so basic renders differently. So you already have issues with whats published in Qiskit in general then in this regard - not good at all if that;s the case.

Reply:
> not good that something so basic renders differently

It's because we rely on the default HTML components for this so the browser takes over the styling. And the styles differ across browsers because ✨ branding ✨

Reply:
Perhaps also environment - I am on Windows, doesn't Mac hide scrollbars by default (its been a while since I touched one)